### PR TITLE
Updated g.json with new mentor details.

### PR DESCRIPTION
**Pull request by gokulchittaranjan**

### CODE REVIEW

File: mentors/python/g.json [hunk 1]
```diff
@@ -15,4 +15,12 @@
         "avatar_url": null,
         "bio": "I'm enthusiastic learner who is always looking for new projects and challenges and a huge Free Software enthusiast. Reach out if you want to talk about technology, life and sports." 
     }
+    {
+        "github_username": "gokulchittaranjan",
+        "name": "Gokul Chittaranjan",
+        "link_text": null,
+        "link_url": "http://github.com/gokulchittaranjan,
```
Comment (SleeplessByte) [inline mentors/python/g.json]:
```suggestion
        "link_url": "http://github.com/gokulchittaranjan"
```

File: mentors/python/g.json [hunk 2]
```diff
@@ -15,4 +15,12 @@
         "avatar_url": null,
         "bio": "I'm enthusiastic learner who is always looking for new projects and challenges and a huge Free Software enthusiast. Reach out if you want to talk about technology, life and sports." 
     }
+    {
+        "github_username": "gokulchittaranjan",
+        "name": "Gokul Chittaranjan",
+        "link_text": null,
+        "link_url": "http://github.com/gokulchittaranjan"
```
Comment (SleeplessByte) [inline mentors/python/g.json]:
```suggestion
        "link_url": "http://github.com/gokulchittaranjan",
```

:boom:

**Comment by SleeplessByte** [comment]
Woops, now the comma is missing ;) @gokulchittaranjan

File: mentors/python/g.json [hunk 3]
```diff
@@ -15,4 +15,12 @@
...
+        "avatar_url": null,
+        "bio": "Have over a deacade of experience working with Python."
```
Comment (rpottsoh) [inline mentors/python/g.json]:
```suggestion
        "bio": "Have over a decade of experience working with Python."
```

File: mentors/python/g.json [hunk 4]
```diff
@@ -15,4 +15,12 @@
         "avatar_url": null,
         "bio": "I'm enthusiastic learner who is always looking for new projects and challenges and a huge Free Software enthusiast. Reach out if you want to talk about technology, life and sports." 
     }
```
Comment (rpottsoh) [inline mentors/python/g.json]:
```suggestion
    },
```
A comma needs to be added here.

**Pull request merged**